remote-api/src/main/java/org/alfresco/rest/api/impl/RenditionsImpl.java

```diff
@@ -107,7 +106,7 @@
     private ServiceRegistry serviceRegistry;
     private ResourceLoader resourceLoader;
     private TenantService tenantService;
-    private RenditionService2 renditionService2;
+    private RenditionService2New renditionService2;
```

Comment:
Rename RenditionService2New - eg. RenditionService3 (that enables switching between options) ?